runelite-mixins/src/main/java/net/runelite/mixins/RSActorMixin.java

```diff
@@ -177,6 +181,16 @@ public Point getMinimapLocation()
 		return Perspective.worldToMiniMap(client, getX(), getY());
 	}
 
+	@Inject
```

Comment:
Huh? `@Inject` on a getter?

Reply:
It's a mixin, so its fields need to be injected.